tensorflow_examples/lite/model_maker/core/task/model_spec/audio_spec.py

```diff
@@ -283,6 +283,8 @@ def _split(self, wav, label):
   ])
   def _preprocess(self, x, label):
     """Preprocess the dataset to extract the spectrum."""
+    # Add small Gaussian noise to the input x to solve the potential "nan" problem of the preprocess_model
```

Comment:
Please wrap this line. Python code is expected to have a max len = 80.

Reply:
Already fixed this problem.